pkgs/top-level/darwin-aliases.nix

```diff
@@ -96,17 +96,17 @@ stubs
 // mapAliases {
   # keep-sorted start case=no numeric=yes block=yes
 
-  bsdmake = pkgs.bmake; # Added 2024-10-03
-  cctools = pkgs.cctools; # Added 2024-07-17
-  cctools-apple = pkgs.cctools; # Added 2024-07-01
-  cctools-llvm = pkgs.cctools; # Added 2024-07-01
-  cctools-port = pkgs.cctools; # Added 2024-07-17
-  discrete-scroll = pkgs.discrete-scroll; # Added 2024-11-27
+  bsdmake = throw "'bsdmake' has been renamed to/replaced by 'bmake'"; # Converted to throw 2025-10-28
+  cctools = throw "'cctools' has been renamed to/replaced by 'cctools'"; # Converted to throw 2025-10-28
+  cctools-apple = throw "'cctools-apple' has been renamed to/replaced by 'cctools'"; # Converted to throw 2025-10-28
+  cctools-llvm = throw "'cctools-llvm' has been renamed to/replaced by 'cctools'"; # Converted to throw 2025-10-28
+  cctools-port = throw "'cctools-port' has been renamed to/replaced by 'cctools'"; # Converted to throw 2025-10-28
+  discrete-scroll = throw "'discrete-scroll' has been renamed to/replaced by 'discrete-scroll'"; # Converted to throw 2025-10-28
   ditto = throw "'darwin.ditto' has been removed, because it was impure and unused"; # Added 2025-10-18
   libresolvHeaders = throw "darwin.libresolvHeaders has been removed; use `lib.getInclude darwin.libresolv`"; # Converted to throw 2025-07-29
   libutilHeaders = throw "darwin.libutilHeaders has been removed; use `lib.getInclude darwin.libutil`"; # Converted to throw 2025-07-29
-  moltenvk = pkgs.moltenvk; # Added 2024-10-06
-  opencflite = pkgs.opencflite; # Added 2024-05-02
+  moltenvk = throw "'moltenvk' has been renamed to/replaced by 'moltenvk'"; # Converted to throw 2025-10-28
+  opencflite = throw "'opencflite' has been renamed to/replaced by 'opencflite'"; # Converted to throw 2025-10-28
```

Comment:
This causes infinite recursion while reviewing.

My best guess is that some of these are not meant to be treated that way.